src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,49 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ title, description, imgUrl, imdbUrl }) => (
+  <>
```

Comment:
do you really need this?

Reply:
oh, didn't notice that div card is a parent element, sorry